error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/StaticImportTest.java

```diff
@@ -99,6 +100,8 @@ void identification() {
             "    // BUG: Diagnostic contains:",
             "    MediaType t2 = MediaType.APPLICATION_JSON;",
             "",
+            "    // BUG: Diagnostic contains:",
+            "    Filters.empty();",
             "    Optional.empty();",
```

Comment:
Tests still pass if I remove line 105, what is the purpose of this line ?
```suggestion
```

Reply:
To show that we are not changing the `Optional.empty()` 😄, so this is expected.

Reply:
I also realise how dumb this comment was 🤦 (Haven't written EPS tests in a while)